### DISCUSSION TITLE
CDN

### DISCUSSION THREAD

**Discussion by avocadowebservices** [DirectoryLister/DirectoryLister#1081] [Ideas & Features]
I would love to see CDN enabled because I use BunnyCDN for my sites. Ideas?

**Comment by PHLAK** [comment]
Directly Lister is intended to be a self-hosted solution to listing files on your server. We have no intention of supporting CDNs or block storage services like S3.

**Comment by avocadowebservices** [comment]
Well CDN is a popular now these days. I think it should be added so I can put files via S3 elsewhere for one reason: Speed

**Reply by PHLAK** [reply]
This is beyond the scope of this project. If you need S3 storage, Directory Lister is not the solution for you.
